src/utils/itemFilterUtils.ts

```diff
@@ -51,27 +51,30 @@ export const buildItemFilterQueryParams = (
   else return ""
 }
 
-export const buildAppliedFiltersString = (
-  query: BibPageQueryParams,
-  numItems = 20,
-  itemAggs: ItemFilterData[]
-) => {
+export const buildItemsString = (query, numItems = 20) => {
```

Comment:
Why is 20 the default?

Reply:
it's just for temporary display purposes because i don't know what kind of data is going to be passed in from the bib page. ill add a comment.